misc/attrib.pm

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/perl
+#
+=head1  NAME
+
+    attrib.pm - two subs to handle ARQ and RRQ message attributes for HPT perl
+    by Stas Mishchenkov 2:460/58.
+
+=head1 SYNOPSIS
+
+   use attrib;
+   
+   sub filter()
+   {
+      is_rrq( $attr );
+   }
+   
+   sub route()
+   {
+      is_arq( $attr, $route );
+   }
+   
+   $attr - message attributes as defined by HPT perl.
+   $route - message routing as defined by HPT perl.
+
+=head1 DESCRIPTION
+
+   This subs to handle ARQ and RRQ message attributes for HPT perl designed
```

Comment:
**These** subs **are** designed to handle ARQ and RRQ message attributes for HPT **Perl**

Please replace "perl" by "Perl" everywhere.